projects/commudle-admin/src/app/feature-modules/external-feed/components/external-feed/external-feed.component.ts

```diff
@@ -10,28 +10,97 @@ import { IFeedItem } from 'projects/shared-models/feed-item.model';
 })
 export class ExternalFeedComponent implements OnInit {
   externalPosts: IFeedItem[] = [];
+  externalPostsV2 = [];
+  page = 1;
+  count = 10;
```

Comment:
Is this of any use? As in if we send the value of count from here then will the server respond accordingly?
If not, then let's remove it

Reply:
Count not needed, removed it.